app/docs/dev/explore/dpp-on-kubernetes.md

```diff
@@ -242,6 +242,11 @@ strings which describe the modifications to be performed.
 
 ### Example
```

Comment:
Just wanted to let you know that for 2.0, this doc moved to `src/production/dp/kubernetes-dp.md` in case you were wondering where it went. I wasn't sure if this was for 2.0 or not.

Reply:
I either don't understand or I cannot find `src/production/dp/kubernetes-dp.md`. Where can I find it?

Reply:
@Automaat It will be on the `release/kuma-2.0` branch.

Reply:
thanks